internal/super/generate.go

```diff
@@ -161,6 +166,19 @@ access(all)
 contract %s {
     init() {}
 }`, name)
+		testFileToWrite = fmt.Sprintf(`import Test
```

Comment:
there seems to be some indentation error here? 

also maybe better we have this string as a constant somewhere outside of this function with a comment on what its supposed to do

Reply:
It actually needs to be like this. If I add a line break the boilerplate file generated will have a line break. But I can move it to a reusable function.